addons/binding/org.openhab.binding.dsmr/src/main/java/org/openhab/binding/dsmr/internal/device/DSMRDevice.java

```diff
@@ -0,0 +1,507 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.dsmr.internal.device;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+
+import org.eclipse.smarthome.core.common.ThreadPoolManager;
+import org.openhab.binding.dsmr.DSMRBindingConstants;
+import org.openhab.binding.dsmr.internal.device.DSMRDeviceConstants.DSMRDeviceEvent;
+import org.openhab.binding.dsmr.internal.device.DSMRDeviceConstants.DSMRPortEvent;
+import org.openhab.binding.dsmr.internal.device.DSMRDeviceConstants.DeviceState;
+import org.openhab.binding.dsmr.internal.device.cosem.CosemObject;
+import org.openhab.binding.dsmr.internal.discovery.DSMRMeterDetector;
+import org.openhab.binding.dsmr.internal.discovery.DSMRMeterDiscoveryListener;
+import org.openhab.binding.dsmr.internal.meter.DSMRMeter;
+import org.openhab.binding.dsmr.internal.meter.DSMRMeterDescriptor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The DSMRDevice class represents the physical Thing within OpenHAB2 context
```

Comment:
remove "within OpenHAB2 context" (or alternatively correct the spelling to "openHAB" ;-))